frontend/components/Table.tsx

```diff
@@ -26,12 +26,12 @@ export function Table(
   return (
     <>
       <div
-        class={`-mx-4 md:mx-0 ring-1 ring-jsr-cyan-950 sm:rounded overflow-hidden ${
+        class={`-mx-4 md:mx-0 ring-1 ring-jsr-cyan-700/30 sm:rounded overflow-hidden ${
```

Comment:
I wouldn't block over this, but here and other places, I'd prefer we use lighter tints of brand colors (editing or adding as necessary) rather than ad-hoc transparency. I just think the probability of inconsistency is a lot higher with non-opaque values.

That is, unless there's a reason we want transparency in these cases, in which case, ignore me.

Reply:
Good point, I ended up changing them to a lighter tint and it was close enough but knowing I have a bunch of these strewn about all over that can become a good guide as to how best to further tweak these colors.